view.rb

```diff
@@ -80,10 +88,25 @@ def self.wrong_entry # Throws error if user enters something that computer doesn
   end
 
   def self.wrong_username_or_pin
```

Comment:
If you open the Ruby Bank, indicate that you have a pre-existing account, you will be then prompted to enter your account name and PIN number. 

If you type in the wrong user name or PIN the code is meant to send you to Dialog::wrong_username_or_pin. It does this and prints the message, but it also shows the following error: 

/view.rb:92:in `wrong_username_or_pin': undefined method `+@' for ",\n":String (NoMethodError)
	from controller.rb:32:in `signin'
	from controller.rb:15:in `signup_signin'
	from controller.rb:129:in `<main>'